src/connect.rs

```diff
@@ -140,11 +124,11 @@ impl KiteConnect {
         let mut resp = self.send_request(url, "POST", Some(data))?;
 
         if resp.status().as_u16() == 200 {
```

Comment:
Can you change this as well to resp.status().is_success()?

Reply:
oh right, missed this. Will update the check!